### ISSUE TITLE
[Brackets LiveDev] Error executing a handler for Runtime.evaluate

### ISSUE THREAD

**Issue by humphd** [humphd/brackets#54]
When I move my mouse over an `<html>` element in the editor, the live dev highlighting code blows up, I assume because it can't find an id for the element.:

```
 Runtime.evaluate
blob:http%3A//localhost%3A8000/fd6e91aa-c647-4742-a746-2d759e246aae:136 [Brackets LiveDev] Error executing a handler for Runtime.evaluate
blob:http%3A//localhost%3A8000/fd6e91aa-c647-4742-a746-2d759e246aae:137 TypeError: Cannot set property 'selector' of null
    at Object.highlightRule (blob:http%3A//localhost%3A8000/fd6e91aa-c647-4742-a746-2d759e246aae:1237:35)
    at eval (eval at <anonymous> (blob:http%3A//localhost%3A8000/fd6e91aa-c647-4742-a746-2d759e246aae:200:41), <anonymous>:1:5)
    at Runtime.evaluate (blob:http%3A//localhost%3A8000/fd6e91aa-c647-4742-a746-2d759e246aae:200:26)
    at blob:http%3A//localhost%3A8000/fd6e91aa-c647-4742-a746-2d759e246aae:133:25
    at Array.forEach (native)
    at Object.MessageBroker.trigger (blob:http%3A//localhost%3A8000/fd6e91aa-c647-4742-a746-2d759e246aae:130:29)
    at Object.ProtocolHandler.message (blob:http%3A//localhost%3A8000/fd6e91aa-c647-4742-a746-2d759e246aae:407:27)
    at blob:http%3A//localhost%3A8000/fd6e91aa-c647-4742-a746-2d759e246aae:47:37
```

Somewhat related to #44 I bet, and how the `<html>` element is being marked in the editor.

**Comment by humphd** [comment]
This is probably something to file upstream, cc @busykai.

**Comment by humphd** [comment]
My guess is that https://github.com/humphd/brackets-browser-livedev/pull/20 will have fixed this.  @gideonthomas, can you confirm?

**Comment by gideonthomas** [comment]
Fixed by 654c4b91017b2af296cdc408d1bffad01d17777b
